src/App.jsx

```diff
@@ -14,33 +18,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD = {
+  alphabeticaly: 'alphabeticaly',
```

Comment:
There is a typo in the key name `alphabeticaly`. It should be `alphabetically` to correctly represent the sorting field.